clang/cmake/caches/Peano-AIE.cmake

```diff
@@ -25,6 +25,7 @@ set(LLVM_ENABLE_PROJECTS
       clang
       clang-tools-extra
       lld
+      libc
```

Comment:
why is this needed?

Reply:
So if IIRC correctly (last week was a long time ago lol), this was needed for `LIBC_TABLEGEN_EXE` to be built. I didn't try to remove it after setting `LLVM_FORCE_BUILD_RUNTIME=ON` so let me try.